src/app/configuration/configuration.component.ts

```diff
@@ -106,7 +106,9 @@ export class ConfigurationComponent implements OnInit {
             'isUserAdmin': true, 'firstName': this.contactFormGroup.value.firstName, 'middleName': this.contactFormGroup.value.middleName,
             'lastName': this.contactFormGroup.value.lastName, 'email': this.contactFormGroup.value.email,
             'phoneNumber': this.contactFormGroup.value.phoneNumber }).subscribe((data1) => {
-             this.planetMessageService.showMessage('Admin created: ' + data1.id.replace('org.couchdb.user:', ''));
+              this.couchService.put('shelf/' + data1.id, { })
```

Comment:
subscribe under subscribe. Can we manage it ? And also don't use data1 , we can use other meaningful variable like result , res and other . Please check it once .